sql/partitioning/partition_base.cc

```diff
@@ -4046,6 +4049,31 @@ bool Partition_base::commit_inplace_alter_table(
   if (ha_alter_info->alter_info->flags == Alter_info::ALTER_PARTITION)
     DBUG_RETURN(false);
 
+  std::string full_table_path = std::string("./") + altered_table->s->db.str +
+                                std::string("/") +
+                                altered_table->s->table_name.str;
```

Comment:
this looks problematic. Better use tablename_to_filename()

Reply:
Good point.. Use table->s->normalized_path.str instead.